backend/editor/entries.py

```diff
@@ -2,24 +2,6 @@
 Database helper functions for API
 """
 import re
-from neo4j import GraphDatabase         # Interface with Neo4J
-from . import settings                  # Neo4J settings
-
-def initialize_db():
-    """
-    Initialize Neo4J database
-    """
-    global driver, session
-    uri = settings.uri
-    driver = GraphDatabase.driver(uri)
-    session = driver.session()
-
-def shutdown_db():
-    """
-    Close session and driver of Neo4J database
-    """
-    session.close()
-    driver.close()

```

Comment:
As said above don't put it neither in entries nor in api, put it in it's own module.